Makefile.js

```diff
@@ -708,6 +736,51 @@ target.checkLicenses = function() {
     });
 };
 
+target.checkGitCommit = function() {
+    var commitMsgs,
+        failed;
+
+    if (hasBranch("master")) {
```

Comment:
Sorry for the flurry here, but I checked a [non-commiter PR Travis build](https://travis-ci.org/eslint/eslint/jobs/71743413), and unless I'm reading it wrong, it appears to indeed pull in master from eslint/eslint.

Not sure if there's really a good way to test this, unless you merge, and then revert if builds start breaking incorrectly.

I'm open to suggestions how to proceed here.